docs/reference/plugins/ant_plugin.rst

```diff
@@ -0,0 +1,97 @@
+.. _rockcraft_ant_plugin:
```

Comment:
what's different in this documentation? Compared to craft-parts'

Reply:
Removed paragraph about /bin/java symlink.